src/main/resources/assets/plethora/lang/nl_be.lang

```diff
@@ -0,0 +1,148 @@
+# Modules
+item.plethora.module.module_introspection.name=Introspectie-module
+item.plethora.module.module_introspection.adjective=Introspectieve
+item.plethora.module.module_introspection.desc=Verpakt je inventaris als randapparatuur. Shift+Gebruik om te linken met speler.
+
+item.plethora.module.module_laser.name=Frickin' laserstraal
+item.plethora.module.module_laser.adjective=Schietend
+item.plethora.module.module_laser.desc=Het schiet lasers. Genoeg gezegd.
+
+death.attack.laser=%1$s was verdampt door %2$s
+death.attack.laser.item=%1$s was verdampt by %2$s met %3$s
+entity.plethora.plethora:laser.name=Een Frickin' laser
+entity.plethora:laser.name=Een Frickin' laser
+
+item.plethora.module.module_scanner.name=Blokscanner
+item.plethora.module.module_scanner.adjective=Scanner
+item.plethora.module.module_scanner.desc=Staat toe dat blokken worden gescand in de buurt
+
+item.plethora.module.module_sensor.name=Entiteitsensor
+item.plethora.module.module_sensor.adjective=Entiteitsensor
+item.plethora.module.module_sensor.desc=Detecteert nabije entiteiten en verzamelt gegevens over hen
+
+item.plethora.module.module_kinetic.name=Kinetische uitbreiding
+item.plethora.module.module_kinetic.adjective=Kinetisch
+item.plethora.module.module_kinetic.desc=Maakt controle over je lichaam mogelijk via de neurale koppeling
+
+item.plethora.module.module_chat.name=Chatrecorder
+item.plethora.module.module_chat.adjective=Luisteren
+item.plethora.module.module_chat.desc=Maakt het luisteren van chat-gebeurtenissen mogelijk
+
+item.plethora.module.module_chat_creative.name=Creative chatrecorder
+item.plethora.module.module_chat_creative.adjective=Onderscheppen
+item.plethora.module.module_chat_creative.desc=Maakt het luisteren en onderscheppen van chat-gebeurtenissen mogelijk
+
+item.plethora.module.module_glasses.name=Overlay-bril
+item.plethora.module.module_glasses.desc=Tekent een overlay over de wereld.
+
+item.plethora.module.module_unknown.name=Onbekende module
+item.plethora.module.module_unknown.desc=Wat heb je deze keer gebroken?
+
+# Manipulator
+tile.plethora.manipulator.name=Manipulator
+tile.plethora.manipulator.mark_1.name=Manipulator Mk I
+tile.plethora.manipulator.mark_2.name=Manipulator Mk II
+tile.plethora.manipulator.mark_1.desc=Wordt gebruikt voor te koppelen met een module vanaf een computer
```

Comment:
Grammar is broken. "te" should be left out.

Reply:
This was changed from `te` to `het` in aba8ec842f78f519d50b7fedf94c1c09f257f9ed. Thoughts?